SimpleMVVM/API Service/Networking.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Networking.swift
+//  SimpleMVVM
+//
+//  Created by Дмитрий Станкевич on 10.02.22.
+//
+
+
+import Foundation
+import UIKit
+
+struct NetworkManager {
+    func fetchImage(code: Int, complition: @escaping (UIImage) -> Void) {
+        var apiImage = UIImage()
+    
+        let session = URLSession.shared
+        let url1 = URL(string: "https://http.cat/" + String(code))!
```

Comment:
please remove all strings constants to `Constants` enum. E.g. 
```
private enum Constants {
 static let baseUrl: String = "https://http.cat/"
}
```
and use it in code